library/analytics/src/main/java/org/cru/godtools/analytics/model/AnalyticsScreenEvent.kt

```diff
@@ -6,10 +6,7 @@ import javax.annotation.concurrent.Immutable
 
 private const val SNOWPLOW_CONTENT_SCORING_URI_PATH_SCREEN = "screenview"
 
-/* Adobe Site Sections */
```

Comment:
this comment shouldn't be removed, the remaining constants are still for adobe site section